params/config.go

```diff
@@ -531,6 +546,8 @@ type TokenOverride struct {
 type Network struct {
 	ChainID                uint64          `json:"chainId"`
 	ChainName              string          `json:"chainName"`
+	DefaultRPCURL          string          `json:"defaultRpcUrl"`      // proxy rpc url
+	DefaultFallbackURL     string          `json:"defaultFallbackURL"` // proxy fallback url
```

Comment:
Wouldn't it be better to call it `ProxyRPCUrl` or smth? 
Especially seeing that you have it `proxy_rpc_url` in the database 🙂

Reply:
It can be proxy, or not. The fact that it is a proxy if a details that can change, in fact from status go point of view, it is a url

Reply:
I mean, we already have `RPCUrl` for each network. `DefaultRPCUrl` is just confusing in this case 🤔 
Anyway, now this is saved as `proxy_rpc_url` into the DB and has a comment `// proxy rpc url`, yet it's called `Default...`